diff --git a/json/json.ml b/json/json.ml
index b1d225a..46efe30 100644
--- a/json/json.ml
+++ b/json/json.ml
@@ -16,18 +16,18 @@ let rec string_of_json json_value =
     begin
       let string_of_obj_pairs pairs =
         let pairs = List.rev_map (fun (key, value) ->
-          Printf.sprintf "\"%s\": %s"
+          Printf.sprintf "\"%s\":%s"
             (String.escaped key)
             (string_of_json value)) pairs
         in
-        String.concat ", " pairs
+        String.concat "," pairs
       in Printf.sprintf "{%s}" (string_of_obj_pairs obj)
     end
   | JsonArray arr ->
     begin
       let string_of_arr_items items =
         let items = List.rev_map string_of_json items in
-        String.concat ", " items
+        String.concat "," items
       in Printf.sprintf "[%s]" (string_of_arr_items arr)
     end
   | JsonString s -> Printf.sprintf "\"%s\"" (String.escaped s)
